plugins/jobs/server/models/job.py

```diff
@@ -151,7 +167,7 @@ def createLocalJob(self, module, function=None, **kwargs):
 
     def createJob(self, title, type, args=(), kwargs=None, user=None, when=None,
                   interval=0, public=False, handler=None, async=False,
-                  save=True, otherFields=None):
+                  save=True, parentId=None, otherFields=None):
```

Comment:
I think a more common pattern is to pass in the parent document rather than just its ID, which removes the need to do a `load` on the parent inside this method.